src/sdk/object_io.js

```diff
@@ -230,10 +229,6 @@ class ObjectIO {
 
             dbg.log0('upload_object: complete upload', complete_params);
 
-            if (params.async_get_last_modified_time) {
```

Comment:
I think this code is not making any sense, `async_get_last_modified_time` is an asynchronous and inline function defined in namespace_cache only. Please correct me if I am wrong.

Although we can add a check for last_modified_time and throws error here, instead of calling this fucntion. wdyt?

Reply:
We should not remove this code, as with this we are waiting for the asynchronous call. Updated the code. Thanks